src/lsp/cobol_preproc/compdir_grammar.mly

```diff
@@ -30,10 +30,11 @@
 %token NE              "<>"         [@keyword (* symbol *) "<>"]
 
 %token CDIR_DEFINE     [@keyword ">>DEFINE", "$DEFINE"]
```

Comment:
There is no `$DEFINE` - see [MF's `>> DEFINE` docs](https://www.microfocus.com/documentation/visual-cobol/vc90/DevHub/GUID-57C303C9-6446-4054-B2DE-2C2A911BCA9B.html).

Instead they have `level 78` and [`$CONSTANT` directive](https://www.microfocus.com/documentation/visual-cobol/vc90/DevHub/HRCDRHCDIR24.html).

Reply:
Thanks for the notice as well.